src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py

```diff
@@ -247,35 +210,23 @@ def web_getPlotImgFromCache(self):
 
         retVal = codeRequestInFileId(fields)
         if not retVal["OK"]:
-            callback = {"success": "false", "error": retVal["Value"]}
-            self.finish(callback)
-            return
+            return {"success": "false", "error": retVal["Value"]}
```

Comment:
```suggestion
            return {"success": "false", "error": retVal["Message"]}
```
But https://github.com/DIRACGrid/DIRAC/blob/c165c81c85f2352aed23baeda1160d6a3f6c5f3c/src/DIRAC/Core/Utilities/Plotting/FileCoding.py#L15 never returns S_ERROR